baking.html

```diff
@@ -31,16 +30,16 @@
         ½ teaspoon ground cinnamon <br>
         1 teaspoon water, or more if necessary</p></div>
         <div class="eight"><h2>Recipe 3</h2></div> 
-        <div class="ten" ><p style="padding: 10px; border: 2px solid rgb(71, 134, 146);">Instrcutions</p></div>
+        <div class="ten" ><p>Instrcutions</p></div>
         <div class="nine"><h2>Recipe 4</h2></div> 
-        <div class="eleven"><p style="padding: 10px; border: 2px solid rgb(71, 134, 146);">Instructions</p></div>
+        <div class="eleven"><p>Instructions</p></div>
 </main>
 <footer>
     <div><h2>Questions or Comments</h2></div><br>
-    <p>Contact me at 
+    <p2>Contact me at 
```

Comment:
`p2` is not a valid html tag

Reply:
I've taken out p2 and corrected the footer appearance. Yes, I did that on purpose because I wanted to change the size.